packages/clay-list/src/ListWithItems.tsx

```diff
@@ -0,0 +1,162 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import classNames from 'classnames';
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClayList from './List';
+import ClaySticker from '@clayui/sticker';
+import React from 'react';
+import {ClayCheckbox} from '@clayui/form';
+import {ClayDropDownWithBasicItems} from '@clayui/drop-down';
+
+interface IListItem {
+	description?: string;
+	dropdownActions?: Array<any>;
+	href?: string;
+	labels?: Array<any>;
+	quickActions?: Array<any>;
```

Comment:
hmm, not sure what you mean here. Can you post a code example?

Reply:
Something similar to this:

```javascript
// my DropDown
const dropdownActions = [
	{
		disabled: true,
		href: '#pageTitle',
		label: 'Download',
		quickAction: true,
		symbolLeft: 'download',
	},
	{
		href: '#pageTitle',
		label: 'Edit',
		symbolLeft: 'edit',
	},
];
```

The `quickAction` prop that defines when it will appear in quick actions is because his idea is to offer a quick action but it will also be listed in DropDown.

Reply:
Hmm, I was under the assumption that dropdown items and quick actions were not necessarily overlapping actions, unless explicitly defined that way. Also, we will have to end up splitting these items up regardless internally, so I think its best to keep them different props.

If we merge them into the same array and prop, we will have to do this sort of thing internally, and sort of defeats the purpose of combining them.
```js
const quickActions = dropdownActions.filter(item => item.quickAction);
const dropdownItems = dropdownActions.filter(item => !item.quickAction);
```

Reply:
> If we merge them into the same array and prop, we will have to do this sort of thing internally, and sort of defeats the purpose of combining them.

We don't need to do that, just do the `dropdownActions` destructuring. Something like that:

```
dropdownActions.map(({quickAction, ...item}) => {
    // render dropdown item
});
```

> What if we didn't want the item to be duplicated in both places?

This should not happen according to Lexicon this is just to get something fast but it should still be listed on DropDown.

Reply:
> This should not happen according to Lexicon this is just to get something fast but it should still be listed on DropDown.

Do you have a link to this? I was browsing the documentation earlier and didn't see this

Reply:
See in `Actions Buttons` it does not look so clear but should be associated, this follows the same idea as the quick actions for the management toolbar.

https://docs.google.com/document/d/1LWHKwKu3LQR6LlJGlXEv8qFzI1EgItYNDWJHnZmBGP0/edit#heading=h.bxhnwo2k3pbz

Reply:
hmm, that definition still doesn't seem too clear. It says that you can use dropdown actions and/or quick actions. Even in our existing clay example, we show quick actions with a different set of dropdown actions. Plus, this doc makes it seem like we would only ever have one quick action, and if there are more then it goes into the dropdown. So it seems like we might need clarification from lexicon on this.

Any idea who would give further clarification on this? cc: @jbalsas

Reply:
hey @drakonux can you clarify this behavior for us?

Reply:
Hey @matuzalemsteles, @bryceosterhaus, so, if I remember correctly, in the old definition, all items needed to be accessible from the dropdown menu. Quick actions would then be duplicated as a button for quick access and as an entry in the dropdown actions.

I'm not sure if we still want to stick to this behaviour, @drakonux?

In any case, I assume we could have 2 different props and just let developers configure this on their own.

Reply:
You are right @jbalsas . We would like to not duplicate quick actions within the dropdown button. The convention on this cases is to not have the same actions, but as Chema said let them configure as they want.

Reply:
Sounds good, thanks @jbalsas and @drakonux. @drakonux will we do the same for the management toolbar?

Reply:
Yep, it's exactly the same for the management toolbar. Thanks @matuzalemsteles